deisctl/units/deis-database.service

```diff
@@ -8,8 +8,9 @@ ExecStartPre=/bin/sh -c "docker inspect deis-database-data >/dev/null 2>&1 || do
 ExecStartPre=/bin/sh -c "IMAGE=`/run/deis/bin/get_image /deis/database` && docker history $IMAGE >/dev/null 2>&1 || docker pull $IMAGE"
 ExecStartPre=/bin/sh -c "docker inspect deis-database >/dev/null 2>&1 && docker rm -f deis-database >/dev/null 2>&1 || true"
 ExecStart=/bin/sh -c "IMAGE=`/run/deis/bin/get_image /deis/database` && docker run --name deis-database --rm --volumes-from=deis-database-data -p 5432:5432 -e EXTERNAL_PORT=5432 -e HOST=$COREOS_PRIVATE_IPV4 $IMAGE"
-ExecStopPost=-/usr/bin/docker exec deis-database sudo -u postgres envdir /etc/wal-e.d/env wal-e backup-push /var/lib/postgresql/9.3/main
-ExecStopPost=-/usr/bin/docker exec deis-database sudo service postgresql stop
+ExecStop=-/usr/bin/docker exec deis-database sudo -u postgres envdir /etc/wal-e.d/env wal-e backup-push /var/lib/postgresql/9.3/main
+ExecStop=-/usr/bin/docker exec deis-database sudo service postgresql stop
+ExecStop=-/usr/bin/docker stop deis-database
```

Comment:
Note that the removed lines were another potential issue. If not for #4412 (which this PR addresses) this would mean the final backup is triggered after the container stops. docker exec would fail at that point.

Since this PR should ensure containers are stopped properly, this would bring the issue to a head, so it's proactively corrected here.